rest/blip_api_test.go

```diff
@@ -1589,3 +1595,47 @@ func TestBlipDeltaSyncPush(t *testing.T) {
 	assert.Equal(t, http.StatusOK, resp.Code)
 	assert.Equal(t, `{"_id":"doc1","_rev":"2-abcxyz","greetings":[{"hello":"world!"},{"hi":"alice"},{"howdy":"bob"}]}`, resp.Body.String())
 }
+
+// TestBlipNonDeltaSyncPush tests that a client that doesn't support deltas can push to a SG that supports deltas (either CE or EE)
+func TestBlipNonDeltaSyncPush(t *testing.T) {
+
+	defer base.SetUpTestLogging(base.LevelTrace, base.KeyAll)()
+	rt := RestTester{DeltaSyncEnabled: base.IsEnterpriseEdition()}
```

Comment:
What about introducing a `NewRestTester()` function as part of this PR which would set the `DeltaSyncEnabled` flag according to `base.IsEnterpriseEdition()`.  We wouldn't necessarily need to convert all the code over to the `NewRestTester()`, but at least new code could leverage it, and maybe at some point we could do a separate PR to refactor all the code.